sys/console/full/src/uart_console.c

```diff
@@ -68,6 +68,7 @@ uart_console_ring_pull_char(struct console_ring *cr)
     uint8_t ch;
 
     ch = cr->buf[cr->tail];
+    cr->buf[cr->tail] = 0xee;
```

Comment:
Does `0xee` get written here just as a debugging aid?

Reply:
oops. that was for debug. I can remove this.